lib/web_ui/test/text/line_breaker_test_helper.dart

```diff
@@ -2,7 +2,6 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-
 // The following test cases contradict rule LB25, so we are replacing them
 // with correct expectations.
 const Map<String, String> _replacements = <String, String>{
```

Comment:
Since these replacements are intentionally contradicting the Unicode rules, should we revise to make sure they don't contain `SPACE (SP) × [13.`? Otherwise, `_checkReplacement` will be contradicting our own contradictions :)